src/kvirc/ui/KviConsoleWindow.cpp

```diff
@@ -396,7 +393,11 @@ void KviConsoleWindow::saveProperties(KviConfigurationFile * cfg)
 
 void KviConsoleWindow::getBaseLogFileName(QString & buffer)
 {
-	buffer = QString("CONSOLE_%1").arg(context()->id());
```

Comment:
> I still need a cup of coffee or 5, but Im not sure that replacing the ID number is a good thing especially since the ID is attached to each context in every other instance that is not log related..

Sorry, I don't understand what you mean here. Can you give a concrete example of a situation where this might be a problem?



> What happens if a specific console ID is reused, see comment in #1710 (comment)

The last hunk is what takes care of that.

> And this still affected by having boolShowMetworkNameForConsoleWindowListEntry on or off?

I don't see how that would affect log naming. What do you mean by "still"?


> is it possible to do this in the same format as channels etc? i.e.

Why would you want the ID?